self-tests fail against tomlkit 0.11.6

With recent tomlkit 0.11.6 `ini2toml` tests fail:
```
...
__________________________________ test_data_files ___________________________________

translator = <ini2toml.translator.Translator object at 0x7fcdedea8e80>

    def test_data_files(translator):
        # Same thing but with the higher level API:
        with pytest.warns(DeprecationWarning, match="'data-files' is deprecated"):
            text = translator.translate(example_data_files, profile_name="setup.cfg")
>           assert text.strip() == expected_data_files.strip()
E           assert '[build-system]\nrequires = ["setuptools>=61.2"]\nbuild-backend = "setuptools.build_meta"\n\n[project]\ndynamic = ["version"]\n\n[tool.setuptools]\ndata-files = {a = ["b"]}\ninclude-package-data = false' == '[build-system]\nrequires = ["setuptools>=61.2"]\nbuild-backend = "setuptools.build_meta"\n\n[project]\ndynamic = ["version"]\n\n[tool]\n[tool.setuptools]\ndata-files = {a = ["b"]}\ninclude-package-data = false'
E               [build-system]
E               requires = ["setuptools>=61.2"]
E               build-backend = "setuptools.build_meta"
E               
E               [project]
E               dynamic = ["version"]
E               
E             - [tool]
E               [tool.setuptools]
E               data-files = {a = ["b"]}
E               include-package-data = false

tests/plugins/test_setuptools_pep621.py:659: AssertionError
...
============================== short test summary info ===============================
FAILED tests/drivers/test_full_toml.py::test_convert - assert '[section1]\n# comment\nvalue = 42 # int value\n\n[section2]\nfloat-value ...
FAILED tests/plugins/test_setuptools_pep621.py::test_split_subtables - assert '[tool.setuptools.packages.find]\nwhere = "src"\n\n[project]\n[project.ent...
FAILED tests/plugins/test_setuptools_pep621.py::test_entrypoints_and_split_subtables - assert '[tool.setuptools.packages.find]\nwhere = "src"\n\n[project]\n[project.ent...
FAILED tests/plugins/test_setuptools_pep621.py::test_empty - assert '[build-system]\nrequires = ["setuptools>=61.2"]\nbuild-backend = "setupto...
FAILED tests/plugins/test_setuptools_pep621.py::test_data_files - assert '[build-system]\nrequires = ["setuptools>=61.2"]\nbuild-backend = "setupto...
============================ 5 failed, 81 passed in 5.75s ============================
```

With https://github.com/sdispater/tomlkit/pull/246 default value for `is_super_table` was changed from `False` to autodetect.

--- a/tests/drivers/test_full_toml.py
+++ b/tests/drivers/test_full_toml.py
@@ -1,4 +1,5 @@
 from ini2toml.drivers import full_toml as lib
+from ini2toml.plugins.profile_independent_tasks import remove_empty_table_headers
 from ini2toml.types import Commented, CommentedKV, CommentedList, CommentKey
 from ini2toml.types import IntermediateRepr as IR
 from ini2toml.types import WhitespaceKey
@@ -27,7 +28,6 @@ b = 2 # 1st line comment
 c = 3
 d = 4 # 2nd line comment
 
-[section3]
 [section3.nested]
 x = "y"
 z = "w" # nested
@@ -64,4 +64,5 @@ example_parsed = IR(
 
 
 def test_convert():
-    assert lib.convert(example_parsed) == example_toml
+    converted = lib.convert(example_parsed)
+    assert remove_empty_table_headers(converted) == example_toml.strip()

--- a/tests/plugins/test_setuptools_pep621.py
+++ b/tests/plugins/test_setuptools_pep621.py
@@ -1,5 +1,6 @@
 import pytest
 
+from ini2toml.plugins.profile_independent_tasks import remove_empty_table_headers
 from ini2toml.plugins.setuptools_pep621 import Directive, SetuptoolsPEP621, activate
 from ini2toml.translator import Translator
 
@@ -175,13 +176,9 @@ django-admin = django.core.management:execute_from_command_line
 """
 
 expected_split_subtables = """\
-[tool]
-[tool.setuptools]
-[tool.setuptools.packages]
 [tool.setuptools.packages.find]
 where = "src"
 
-[project]
 [project.entry-points]
 # For example:
 
@@ -197,7 +194,8 @@ def test_split_subtables(plugin, parse, convert):
     doc = plugin.split_subtables(doc)
     print("@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@")
     print(doc)
-    assert convert(doc).strip() == expected_split_subtables.strip()
+    converted = convert(doc).strip()
+    assert remove_empty_table_headers(converted) == expected_split_subtables.strip()
 
 
 # ----
@@ -217,13 +215,9 @@ pyscaffold.cli =
 """
 
 expected_entrypoints_and_split_subtables = """\
-[tool]
-[tool.setuptools]
-[tool.setuptools.packages]
 [tool.setuptools.packages.find]
 where = "src"
 
-[project]
 [project.entry-points]
 # For example:
 
@@ -248,7 +242,9 @@ def test_entrypoints_and_split_subtables(plugin, parse, convert):
     doc = plugin.split_subtables(doc)
     print("@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@")
     print(doc)
-    assert convert(doc).strip() == expected_entrypoints_and_split_subtables.strip()
+    converted = convert(doc).strip()
+    expected = expected_entrypoints_and_split_subtables.strip()
+    assert remove_empty_table_headers(converted) == expected
 
 
 # ----
@@ -607,7 +603,6 @@ build-backend = "setuptools.build_meta"
 [project]
 dynamic = ["version"]
 
-[tool]
 [tool.setuptools]
 include-package-data = false
 """
@@ -621,7 +616,8 @@ def test_empty(translator, plugin, parse, convert):
     doc = plugin.pep621_transform(doc)
     print("@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@")
     print(doc)
-    assert convert(doc).strip() == expected_empty.strip()
+    converted = convert(doc).strip()
+    assert remove_empty_table_headers(converted) == expected_empty.strip()
 
     # Same thing but with the higher level API:
     text = translator.translate("", profile_name="setup.cfg")
@@ -645,7 +641,6 @@ build-backend = "setuptools.build_meta"
 [project]
 dynamic = ["version"]
 
-[tool]
 [tool.setuptools]
 data-files = {a = ["b"]}
 include-package-data = false
@@ -656,7 +651,7 @@ def test_data_files(translator):
     # Same thing but with the higher level API:
     with pytest.warns(DeprecationWarning, match="'data-files' is deprecated"):
         text = translator.translate(example_data_files, profile_name="setup.cfg")
-        assert text.strip() == expected_data_files.strip()
+        assert remove_empty_table_headers(text.strip()) == expected_data_files.strip()
 
 
 # ----

